src/actors/player.gd

```diff
@@ -148,12 +166,16 @@ func move_state():
 	var input_vector = Vector2.ZERO
 	if Input.is_action_pressed("right"):
 		input_vector.x += 1
+		set_idle_timer()
 	if Input.is_action_pressed("left"):
 		input_vector.x -= 1
+		set_idle_timer()
 	if Input.is_action_pressed("down"):
 		input_vector.y += 1
+		set_idle_timer()
 	if Input.is_action_pressed("up"):
 		input_vector.y -= 1
+		set_idle_timer()
```

Comment:
Seems like we could call `set_idle_timer()` after all the inputs are processed, instead of once in every condition.